training_system/app/views/trainer/courses/edit.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="container-fluid mt--6">
+  <div class="row">
+    <div class="col-xl-8 order-xl-1">
+      <div class="card">
+        <div class="card-header">
+          <div class="row align-items-center">
+            <div class="col-8">
+              <h3 class="mb-0"><% t("views.course.edit.header_edit") %></h3>
+            </div>
+          </div>
+        </div>
+        <div class="card-body">
+          <%= form_for [:trainer, @course], class: "form-group" do |f| %>
+
+            <%= f.label :name %>
+            <%= f.text_field :name, class: "form-control" %>
+
+            <%= f.label :description %>
+            <%= f.text_field :description, class: "form-control" %>
+
+            <%= f.label :start_date %>
+            <%= f.date_field :start_date, class: "form-control" %>
+
+            <%= f.label :end_date, "End Date" %>
```

Comment:
I18n chú

tìm hiểu cái I18n với model nhé
để ko cần gọi ở view 
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

Reply:
dạ chị